docs/CODE_OF_CONDUCT.rst

```diff
@@ -0,0 +1,127 @@
+.. _plasmapy-code-of-conduct:
```

Comment:
`checking consistency... /home/circleci/project/docs/CODE_OF_CONDUCT.rst: WARNING: document isn't included in any toctree`